<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultDialog</class>
 <widget class="QDialog" name="ResultDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RESULTS</string>
   </property>
  </widget>
  <widget class="QLabel" name="Player1Label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Player2Label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="Player3Label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="Player4Label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="Player1PointsLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>POINTS</string>
   </property>
  </widget>
  <widget class="QLabel" name="Player2PointsLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>POINTS</string>
   </property>
  </widget>
  <widget class="QLabel" name="Player3PointsLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>POINTS</string>
   </property>
  </widget>
  <widget class="QLabel" name="Player4PointsLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>POINTS</string>
   </property>
  </widget>
  <zorder>Player4PointsLabel</zorder>
  <zorder>closeButton</zorder>
  <zorder>label</zorder>
  <zorder>Player1Label</zorder>
  <zorder>Player2Label</zorder>
  <zorder>Player3Label</zorder>
  <zorder>Player4Label</zorder>
  <zorder>Player1PointsLabel</zorder>
  <zorder>Player2PointsLabel</zorder>
  <zorder>Player3PointsLabel</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ResultDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
